db/table_cache.cc

```diff
@@ -171,6 +171,7 @@ Status TableCache::FindTable(const ReadOptions& ro,
 
   if (*handle == nullptr) {
     if (no_io) {  // Don't do IO and return a not-found status
+      s.PermitUncheckedError();
```

Comment:
Can you fix the comment above about "Not Found',  It returns Incomplete"

Reply:
Done.